### DISCUSSION TITLE
Geometry Dash

### DISCUSSION THREAD

**Discussion by CallMeEchoCodes** [catppuccin/catppuccin#2054] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Geometry Dash has its textures as flat PNGs. To texture pack it all you do is edit those PNGs. This is complicated by each sprite having 3 resolutions it needs to be in and all being in spritesheets however programs exist to split them into just textures.

### Any additional comments?

This is a pretty easy port for the most part. Just a really really big coloring sheet.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by Aziz18413** [comment]
Hello! It's been 2 years and I want my GD to have a Catppuccin theme to match my whole PC. How's the TP going?

**Comment by scarcekoi** [comment]
Currently working on one here: https://github.com/scarcekoi/geometry-dash

**Comment by Jotalea** [comment]
I think this texture pack (GDClean) would look pretty nice with the Catppuccin color palette.

https://github.com/Asky740/gdclean
